tagstudio/src/qt/modals/drop_import.py

```diff
@@ -167,10 +167,11 @@ def duplicates_choice(self) -> int:
         dupes_to_show = self.duplicate_files
         if len(self.duplicate_files) > display_limit:
             dupes_to_show = dupes_to_show[0:display_limit]
+        
+        dupes_str = '\n    '.join(map(lambda path: str(path),dupes_to_show))
+        dupes_more = f"\nand {len(self.duplicate_files)-display_limit} more " if len(self.duplicate_files) > display_limit else '\n'
```

Comment:
But now there's a backslash inside of the f-string again, causing the issue in #289 😅

Reply:
That's a backslash inside the template, not the expression. Python was upset about backslashes being between the `{ }` curly brackets. It seems to be running for me with these changes on 3.11.2

Reply:
Ah interesting, in that case if it works then I've got no issue with it